packages/assets-controllers/src/NftController.ts

```diff
@@ -507,16 +508,21 @@ export class NftController extends BaseController<
    *
    * @param contractAddress - Hex address of the NFT contract.
    * @param tokenId - The NFT identifier.
+   * @param networkClientId - The networkClientId identifier.
    * @returns Promise resolving to the current NFT name and image.
    */
   async #getNftInformationFromApi(
     contractAddress: string,
     tokenId: string,
+    networkClientId?: NetworkClientId,
```

Comment:
Nit: Since this is an optional argument what are your thoughts on placing this within an options bag? This would allow more optional arguments to be added in the future without worrying about passing them in a specific order.

``` suggestion
    { networkClientId }: {
      networkClientId?: NetworkClientId,
    }